ZenPacks/zenoss/WBEM/modeler/wbem.py

```diff
@@ -186,3 +207,25 @@ def handle_failure(failure):
     deferred.addErrback(handle_failure)
 
     return deferred
+
+
+def check_if_complete(results, log, device, namespace, classname, _results=None):
+    if not _results:
+        _results = []
+
+    if isinstance(results, tuple):
+        _results.append(results[0])
+        credentials = (device.zWBEMUsername, device.zWBEMPassword)
+        context = (namespace, results[2])
+
+        wbemClass = PullInstances(credentials, context, device.zWBEMMaxObjectCount, classname)
```

Comment:
The line is longer than 79 characters.

Reply:
79 is a legacy senseless constraints, originally this module uses 120 symbols limit.